### ISSUE TITLE
ReferenceError when using devtools middleware

### ISSUE THREAD

**Issue by henqx** [react-spring/zustand#55]
Hi, I'm getting a `ReferenceError: Cannot access 'state' before initialization` when using zustand(1.0.4) with the devtools middleware in a create-react-app based setup. The error is only present in development mode.

To reproduce:
1. `npx create-react-app test-zustand && cd test-zustand`
2. `yarn`
3. `yarn add zustand`
4. Replace App.js with [TestApp]
3. `yarn start`

[TestApp]:
```
import React from "react";
import "./App.css";
import create from "zustand";
import { devtools } from "zustand/middleware";

const [useStore] = create(
  devtools(set => ({
    count: 1,
    inc: () => set(state => ({ count: state.count + 1 })),
    dec: () => set(state => ({ count: state.count - 1 }))
  }))
);

function App() {
  const count = useStore(state => state.count);
  const inc = useStore(state => state.inc);
  return (
    <div className="App">
      <header className="App-header">
        <p>Count: {count}</p>
        <button onClick={inc}>Inc</button>
      </header>
    </div>
  );
}

export default App;
```

Screenshot:
![Screenshot from 2019-08-20 09-44-37](https://user-images.githubusercontent.com/16530091/63323718-28fc1880-c32f-11e9-8e67-930bb9ec6f6a.png)

**Comment by drcmda** [comment]
im still on vacation, wont be able to look into it until september. @JeremyRH would you take a quick look? can it be that the batchedupdates update caused it?

**Comment by JeremyRH** [comment]
Should be fixed in 1.0.7.

**Comment by henqx** [comment]
Thank you. Updating to 1.0.7 fixed the error for me.
